modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
+
+* With this update, the sample pipelines displayed in the web console use the `v1` version of the {pipelines-shortname} API.
+
+[id="cli-new-features-1-16_{context}"]
+=== CLI
+
+* With this update, you can use the `tkn customrun delete <custom_run_names>` command to delete one or more custom runs.
+
+* With this update, when you run a `tkn <resource> list` command with the `-o` YAML flag, the listed resources are now separated with `---` separators to enhance readability of the output.
+
+[id="pac-new-features-1-16_{context}"]
+=== {pac}
+
+* With this update, if you create two `PipelineRun` definitions with the same name, {pac} logs an error and does not run either of these pipeline runs.
+
+* With this update, the {pac} `pipelines_as_code_pipelinerun_count` metric allows filtering of the `PipelineRun` count by repository or namespace.
+
+* With this update, the `readOnlyRootFilesystem` security context for the {pac} controller, webhook, and watcher is set to `true` by default to increase security and avoid being flagged by the security scanner.
+
+[id="tekton-chains-new-features-1-16_{context}"]
+=== {tekton-chains}
+
+* With this update, when using `docdb` storage in {tekton-chains}, you can configure the `MONGO_SERVER_URL` value directly in the `TektonConfig` CR as the `storage.docdb.mongo-server-url` setting. Alternatively, you can provide this value using a secret and set the `storage.docdb.mongo-server-url-dir` setting to the directory where the `MONGO_SERVER_URL` file is located.
++
+.Example of creating a secret with the `MONGO_SERVER_URL` value
+[source,terminal]
+----
+$ oc create secret generic mongo-url -n tekton-chains \ #
+  --from-file=MONGO_SERVER_URL=/home/user/MONGO_SERVER_URL 
+----
++
+.Example of configuring the `MONGO_SERVER_URL` value using a secret
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  chain:
+    disabled: false
+    storage.docdb.mongo-server-url-dir: /tmp/mongo-url
+    options:
+      deployments:
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: tekton-chains-controller
+                  volumeMounts:
+                  - mountPath: /tmp/mongo-url
+                    name: mongo-url
+                volumes:
+                -  name: mongo-url
+                   secret:
+                    secretName: mongo-url
+# ...
+----
+
+* With this update, when using KMS signing in {tekton-chains}, instead of providing the KMS authentication token value directly in the configuration, you can provide the token value as a secret by using the `signers.kms.auth.token-path` setting.  
++
+[source,terminal]
```

Comment:
Add a title to this code sample? I notice you do this in most all of the examples, unless I've glazed past some 😆 Its nice and kinda ties back to the bullet clearly.

Reply:
Very good point, @eromanova97 please do it, and yeah it was I who missed this.

Reply:
But I definitely forgot to replace the title with an introductory sentence for the command. Something like: `To create a KMS token secret, run the following command:`

Reply:
Yes, please go ahead with this.